src/Paprika/Data/SlottedArray.cs

```diff
@@ -450,7 +469,7 @@ private Span<byte> GetSlotPayload(ref Slot slot)
     /// </summary>
     public static ushort HashForTests(in NibblePath key) => Slot.PrepareKey(key, out _, out _);
 
-    [StructLayout(LayoutKind.Explicit, Size = Size)]
+    [StructLayout(LayoutKind.Sequential, Pack = sizeof(byte), Size = Size)]
     private struct Slot
```

Comment:
This is persistent data structure. I believe we should go with `Explicit` here. We might come up with an attribute for such cases or a comment.